extensions-contrib/confluent-extensions/pom.xml

```diff
@@ -17,7 +17,7 @@
   <parent>
     <artifactId>druid</artifactId>
     <groupId>org.apache.druid</groupId>
-    <version>24.0.0-SNAPSHOT</version>
+    <version>24.0.0</version>
```

Comment:
I just pushed this commit to fix this on the base branch: 
https://github.com/confluentinc/druid/commit/aededec90808f453547874791d9d41940da7d4e1